de-DE/Codex_der_Charaktere/02_Wie_man_spielt/Wie_man_spielt.md

```diff
@@ -1,4 +1,159 @@
 # Wie man spielt
 
-> **Warnung:**
-> Dieser Abschnitt wird noch übersetzt.
+Ein Pen-&-Paper-Rollenspiel (TTRPG) zu spielen bedeutet, sich eine Figur in einer Fantasiewelt vorzustellen und mit Würfeln zu bestimmen, was geschieht.  
+So funktioniert es:
+
+**Wähle eine Spielleitung.**  
+Eine Person ist die Spielleitung. Sie denkt sich die Geschichte aus, kontrolliert die Welt und beschreibt Alles und Alle um dich herum.  
+\
+**Erschaffe deine Spielfigur.**  
+Wähle eine Klasse, eine Abstammung, eine Kultur und einen Hintergrund für deine Figur.  
+Das ist dein*e Held*in in der Spielwelt!  
+\
+**Eine Begegnung spielen:**  
+In deinem Zug beschreibst du, was dein Charakter tut.  
+Schwingst du dein Schwert, wirkst du einen Zauber oder sprichst du mit jemandem?  
+Um herauszufinden, ob du Erfolg hast, würfelst du einen zwanzigseitigen Würfel (W20) und addierst deinen Attributsmodifikator und eventuell einen Fertigkeitsbonus.  
+Höhere Würfe sind besser!
+
+> **Grau (Spielleitung):**  
```

Comment:
Either the person continues to be called "Grey" (which is a name), or we rename them; but "Grau" not a person's name. 😅